Update orchestrion references in Cloud Run and Container App docs (#19212)
DataDog/documentation: content/en/serverless/azure_container_apps/_index.md

{{< /programming-lang >}}
{{< programming-lang lang="go" >}}

[Manually install][1] the Go tracer before you deploy your application. Add the following instructions and arguments to your Dockerfile.

```
COPY --from=datadog/serverless-init:1 /datadog-init /app/datadog-init
ENTRYPOINT ["/app/datadog-init"]
ENV DD_SERVICE=datadog-demo-run-go
ENV DD_ENV=datadog-demo
ENV DD_VERSION=1
CMD ["/path/to/your-go-binary"]
```

#### Explanation

1. Copy the Datadog `serverless-init` into your Docker image.
   ```
   COPY --from=datadog/serverless-init:1 /datadog-init /app/datadog-init
   ```

> 2. Change the entrypoint to wrap your application into the Datadog `serverless-init` process

Reviewer: ```suggestion
2. Change the entrypoint to wrap your application into the Datadog `serverless-init` process.
```

   ```
   ENTRYPOINT ["/app/datadog-init"]
   ```

3. (Optional) Add Datadog tags.
   ```
   ENV DD_SERVICE=datadog-demo-run-go
   ENV DD_ENV=datadog-demo
   ENV DD_VERSION=1
   ```

4. Execute your binary application wrapped in the entrypoint. Adapt this line to your needs.
   ```
   CMD ["/path/to/your-go-binary"]
   ```

#### Orchestrion

> **Note**: [Orchestrion][2] is a tool for automatically instrumentation of Go code, which is currently in Private Beta. With Orchestrion, it is possible to instrument your Go applications via Dockerfile instruction. If you are interested in participating in the Beta or providing feedback on Orchestrion, please open a Github issue or reach out to support. 

Reviewer: ```suggestion
**Note**: [Orchestrion][2] is a tool for automatically instrumenting Go code, which is currently in Private Beta. With Orchestrion, it is possible to instrument your Go applications through Dockerfile. If you are interested in participating in the Beta or providing feedback on Orchestrion, please open a Github issue or reach out to support. 
```
